core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -169,10 +178,18 @@
                                    gracefulShutdownQuietPeriod, "gracefulShutdownQuietPeriod");
 
         requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
-        blockingTaskExecutor =
-                ExecutorServiceMetrics.monitor(meterRegistry, blockingTaskExecutor,
-                                               "blockingTaskExecutor", "armeria");
+        if (!(blockingTaskExecutor instanceof ThreadPoolExecutor ||
+              blockingTaskExecutor instanceof ForkJoinPool)) {
+            logger.warn("Cannot record metrics with {}. (expected: ThreadPoolExecutor or ForkJoinPool)",
+                        blockingTaskExecutor);
+        }
+        ExecutorServiceMetrics.monitor(meterRegistry, ((BlockingTaskExecutor) blockingTaskExecutor).unwrap(),
+                                       "blockingTaskExecutor", "armeria");
+        blockingTaskExecutor = new TimedScheduledExecutorService(meterRegistry, blockingTaskExecutor,
+                                                                 "blockingTaskExecutor", "armeria",
```

Comment:
`armeria.` was not a type. 😉 See https://github.com/micrometer-metrics/micrometer/blob/1e797415edb110e01214257e035c3d1b37b7e1e2/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java#L269
```suggestion
                                                                 "blockingTaskExecutor", "armeria.",
```
You

Reply:
Thank you!